### DISCUSSION TITLE
[HELP] Cómo puedo configurar un campo de fecha para que los valores mínimos y máximos sean relativos a la fecha actual?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#209] [2. Questions and Answers] [answered]
Si está mal redactado el título favor su ayuda para arreglar de tal manera que se entienda con la descripción que dejaré a continuación.

Resulta que se debe enviar una solicitud con 7 días de anticipación desde el día que envío el requerimiento (por ejemplo: si quiero utilizar  una sala de eventos el día 1 de Julio, la debo pedir hoy, y si la pido mañana el sistema me debe arrojar un mensaje indicando que no se puede para esa fecha).

La otra opción es que me saliera los días "inhabilitado" o gris y solo pudiera ingresar los días despues de 7 días.

Traté en usar Tipo de max/min y no tuve buen resultado (quizás por la falta de información en cómo se debe configurar).

Gracias :)

**Comment by guillevalin** [comment]
@NicolasPLG para ello puedes usar el subtype "relative" de la pregunta de fecha (survey+datetime).

![image](https://user-images.githubusercontent.com/34345309/119692036-10c8ff80-be19-11eb-8264-15b90195f429.png)

Tienes dos formatos "relativos", el relacionado a la fecha de creación del formulario, y el segundo relativo a la fecha actual, probablemente el caso que te interesa es el segundo.

Luego de eso, tu defines min y max respecto a la fecha actual, en segundos.

**Reply by NicolasPLG** [reply]
hola @guillevalin creo que no me funciona, probé las 2 formas de cada tiempo relativo y no obtengo el resultado que quiero, quizás estoy cometiendo un error. Dejo un video de los casos cuando lo configuro.

Mi objetivo es: si levanto una solicitud hoy 26-05-2021, no me debe aceptar las fechas, 26,27,28,29,30,31,1 que son 7 días. Por lo que si selecciono una fecha para el 2 si me permitiría

Adjunto el link video: https://drive.google.com/file/d/16O9E1vzgSQqsikUT-F8N3afnaT_0KBV1/view?usp=sharing

plz gracias :>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Tienes que configurarlo de la siguiente manera
![image](https://user-images.githubusercontent.com/34345309/119701642-9604e200-be22-11eb-91b5-5d8eef3e249d.png)

Este ejemplo valida que la fecha sea +/- 1 día desde la fecha de envío:
![image](https://user-images.githubusercontent.com/34345309/119701742-b3d24700-be22-11eb-948a-1dad35bc5dee.png)

**Reply by NicolasPLG** [reply]
@guillevalin pero si colocas 27-05-2021, cómo se hace para que arroje el mensaje? :(

No entiendo mucho el maximo :(

**Reply by guillevalin** [reply]
@NicolasPLG Si pongo 27/05 funciona, pero con 28/05 tampoco funciona:

![image](https://user-images.githubusercontent.com/34345309/119724216-7084d200-be3c-11eb-8981-bd3ba203e660.png)

![image](https://user-images.githubusercontent.com/34345309/119724250-7b3f6700-be3c-11eb-978a-52533492e825.png)

La opción "Relative to answered time" evalúa respecto a la fecha de cuando estás respondiendo (26-05) y está configurado para +/- 1 día.

**Reply by NicolasPLG** [reply]
Entonces se comporta de la siguiente manera:
![image](https://user-images.githubusercontent.com/61419471/119735988-1d664b80-be4b-11eb-8eff-f1a46aeca9cd.png)

Pero lo que ando buscando es esto, según lo que entendí cómo funcionan los mínimos y máximos
![image](https://user-images.githubusercontent.com/61419471/119736033-28b97700-be4b-11eb-9d43-c7d4b56145e5.png)

Será posible hacer eso?

Gracias :>

**Reply by guillevalin** [reply]
@NicolasPLG Según lo que te entiendo, solo quieres fechas 7 días en el futuro de la fecha actual. Para hacer esto esta configuración también es válida:

![image](https://user-images.githubusercontent.com/34345309/119845293-23097300-bed7-11eb-98b2-20ba4babcc55.png)

Lo que te permitiría valores válidos desde 7 días en el futuro, hasta un año en el futuro. Claramente 365 es un valor arbitrario que podría ser mayor o menor dependiendo de tus necesidades.

**Reply by NicolasPLG** [reply]
funcionó como me mencionaste

gracias @guillevalin
